ragstack-e2e-tests/e2e_tests/llama_index/test_astra.py

```diff
@@ -220,6 +221,39 @@ def test_vector_search_with_metadata(environment: Environment):
     # documents = index.as_retriever().retrieve("RAGStack")
     # assert len(documents) == 0
 
+def test_llamaparse_with_vector_search(environment: Environment):
```

Comment:
This is not the right file, test_astra is about testing astra capabilities, we can start a new file in the llama index directory